src/asset-swapper/swap_quoter.ts

```diff
@@ -206,16 +194,17 @@ export class SwapQuoter {
             }),
             exchangeProxyOverhead: (flags) => gasPrice.times(opts.exchangeProxyOverhead(flags)),
         };
+
         // pass the rfqClient on if rfqt enabled
         if (calcOpts.rfqt !== undefined) {
             calcOpts.rfqt.quoteRequestor = new QuoteRequestor();
             calcOpts.rfqt.rfqClient = rfqClient;
         }
-
-        const result: OptimizerResultWithReport = await this._marketOperationUtils.getOptimizerResultAsync(
+        const limitOrders = await this.getLimitOrders(marketOperation, makerToken, takerToken, calcOpts);
```

Comment:
how's the test coverage on this? i see we have a unit test for "should respect excludedSources" which should cover this?

Reply:
"should respect excludedSources" does something else imo so i renamed it... 👀 

I found a compromised way testing this (better than not having any coverage).
Ideally the `swap_test.ts` should support at least one non-`Native` source... 😣